sdk/iothub/azure-iot-deviceprovisioning/azure/iot/deviceprovisioning/aio/operations/_patch.py

```diff
@@ -0,0 +1,875 @@
+# ------------------------------------
```

Comment:
When I look at the commit in APIview it shows query returning `ItemPaged[JSON]` and no x_ms_continuation or x_ms_max_item_count parameters (expected) but multiple overloads of the existing non-optimized operation are exposed. Is this intended?

![image](https://github.com/Azure/azure-sdk-for-python/assets/28658112/ff7e55af-104d-409b-8438-03316a485a42)